src/Spice86.Core/Emulator/Devices/ExternalInput/Pic.cs

```diff
@@ -119,8 +119,8 @@ public class Pic {
         bool sendEndOfInterruptCommand = (value & 0b10_0000) != 0;
         bool sendSpecificCommand = (value & 0b100_0000) != 0;
         bool rotatePriorities = (value & 0b1000_0000) != 0;
-        if (_loggerService.IsEnabled(Serilog.Events.LogEventLevel.Information)) {
-            _loggerService.Information(
+        if (_loggerService.IsEnabled(Serilog.Events.LogEventLevel.Verbose)) {
```

Comment:
Some games spam the pic a lot of times per seconds, are you sure this is not too impacting on the logs verbosity?

Reply:
This change actualy _reduces_ the amount of logging.
But I did not intend to commit this to this branch, so I have reverted it.